Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -18,24 +18,59 @@ import Foundation
 
 // MARK: QueryBuilder
 
-/// Used in code dealing with variances between the various database engines. As
-/// different databases have different query syntax, sometimes changes need to be
-/// made when generating the actual SQL statement to run. Additional needed changes
-/// should be done by updating QueryBuilder substitutions array. Every query component
-/// then builds its string representation using that array.
+/**
+ Note: Changing the `QueryBuilder` should only be needed for adding support for a new database plugin.
+ 
+ `QueryBuilder` is used in code dealing with variances between the various database engines. As
+ different databases have different query syntax, sometimes changes need to be
+ made when generating the actual SQL statement to run. Additional needed changes
+ should be done by updating `QueryBuilder` substitutions array. Every query component
```

Comment:
"Additional needed changes should be done by updating `QueryBuilder` substitutions array" 
please can you change this to 
"Additional changes should be made by updating the `QueryBuilder` substitutions array."